src/Illuminate/Database/Query/Expression.php

```diff
@@ -2,19 +2,22 @@
 
 namespace Illuminate\Database\Query;
 
-class Expression
+use Illuminate\Contracts\Database\Query\Expression as ExpressionContract;
+use Illuminate\Database\Grammar;
+
+class Expression implements ExpressionContract
 {
     /**
      * The value of the expression.
      *
-     * @var mixed
+     * @var scalar
```

Comment:
Please use the proper types here instead of "scalar". Do this for all other occurrences in this PR.

Reply:
Done